packages/wp-source/src/libraries/redirect.ts

```diff
@@ -0,0 +1,23 @@
+import { getMatch } from "./get-match";
+import WpSource from "../..";
+
+export const redirect = (
+  path: string,
+  redirections: WpSource["libraries"]["source"]["redirections"]
+) => {
+  // sort redirections
+  redirections.sort(({ priority: p1 }, { priority: p2 }) => p1 - p2);
+
+  // apply redirections in order
+  let lastPriority = -1;
```

Comment:
Uhm... what happens if people use negative priorities?

Reply:
It  won't work... But `lastPriority` could have a value of  `-Infinity`. I'll change that.

Reply:
Maybe we should do the same for the other arrays with priorities? (handlers and processors)